bin/run_ess_sensors.py

```diff
@@ -21,12 +21,40 @@
 import asyncio
 import logging
 
-from lsst.ts.ess import sensors
+from lsst.ts.ess.sensors import SocketServer
 
 logging.basicConfig(
     format="%(asctime)s:%(levelname)s:%(name)s:%(message)s",
     level=logging.DEBUG,
 )
 
 
-asyncio.run(sensors.SocketServer.main())
+async def main():
+    """Main method that, when executed in stand alone mode, starts the socket
+    server.
+
+    The SocketServer automatically stops once the client exitst and at that
```

Comment:
`exitst` -> `exits` (I think; surely you didn't mean `exists`)

Reply:
Thanks for catching that. Yes, I meant `exits`. Done.